### PR TITLE
Move Smart contracts to Besu Library

### PR DESCRIPTION
**Pull request by shemnon**
**Description**:

HIP describing moving the Smart Contract library to Besu and the
secondary implications such as London hard fork support.

Signed-off-by: Danno Ferrin <danno.ferrin@hedera.com>

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-smart-contract-besu.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
+- hip:
```
Comment (sergmetelin) [inline HIP/hip-smart-contract-besu.md:1]: Please assign HIP-26 to this HIP document and rename the file to `hip-26.md`

File: HIP/hip-smart-contract-besu.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+- title: Migrate Smart Contract Service EVM to Hyperledger Besu EVM
+- author: Daniel Ivanov (daniel@limechain.tech), Danno Ferrin (
+  danno.ferrin@hedera.com)
+- type: Standards
+- category: Services
+- status: Draft
+- created: 16 September 2021
+- discussions-to:
```
Comment (sergmetelin) [inline HIP/hip-smart-contract-besu.md:9]: Please link to this Discussion thread: https://github.com/hashgraph/hedera-improvement-proposal/discussions/140

File: HIP/hip-smart-contract-besu.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+- updated:
```
Comment (sergmetelin) [inline HIP/hip-smart-contract-besu.md:10]: Please update with the relevant date

**Review by sergmetelin — CHANGES_REQUESTED** [review]

File: HIP/hip-smart-contract-besu.md (hunk 1 above)
Comment (shemnon) [reply HIP/hip-smart-contract-besu.md]: Done

File: HIP/hip-smart-contract-besu.md (hunk 2 above)
Comment (shemnon) [reply HIP/hip-smart-contract-besu.md]: done

File: HIP/hip-smart-contract-besu.md (hunk 3 above)
Comment (shemnon) [reply HIP/hip-smart-contract-besu.md]: done

**Review by sergmetelin — DISMISSED** [review]
LGTM

**Review by sergmetelin — APPROVED** [review]
Do you mind renaming

**Pull request merged**